app/main.ipynb

```diff
@@ -336,43 +440,1070 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "source": [
+    "# Python code\n",
+    "%time [i for i in range(1000000)]"
```

Comment:
In the numpy version, you should use `np.arange(1000000)` directly without wrapping it with `np.array`, as `np.arange` already returns a numpy array.